ui/src/containers/Details/index.tsx

```diff
@@ -6,16 +6,29 @@ import { useMst } from '../../store/root';
 import BasicDetails from '../BasicDetails';
 import Description from '../../components/Description';
 import { assert } from '../../store/utils';
+import { NotFound } from '../../components/PageNotFound';
+import { titleCase } from '../../common/titlecase';
+import { ICatalog } from '../../store/catalog';
 
 const Details: React.FC = () => {
   const { resources, user } = useMst();
-  const { name } = useParams();
+  const { name, catalog, kind } = useParams();
+
+  const catalogs = resources.catalogs.values;
+  const validateUrl = () => {
+    let catalogUrl: boolean | undefined;
```

Comment:
can't we just do this like
```suggestion
    let catalogUrl = false;
```